src/TeachMeSkills/Homework/Homework2.java

```diff
@@ -0,0 +1,144 @@
+        System.out.print("Введите номер месяца, чтобы узнать пору года: ");
+        month = sc.nextInt();
+        if (month == 12 || month == 1 || month == 2) {
+            season = "Зима";
+        } else if (month == 3 || month == 4 || month == 5) {
+            season = "Весна";
+        } else if (month == 6 || month == 7 || month == 8) {
+            season = "Лето";
+        } else if (month == 9 || month == 10 || month == 11) {
+            season = "Осень";
+        } else {
+            season = "Введённое число не является номером месяца";
+        }
+        System.out.println(season);
+
+        // Управляющие конструкции. Задание 3
+        System.out.print("Введите число: ");
+        int number = sc.nextInt();
+        if (number % 2 == 0) {
+            System.out.println("Введённое число чётное");
+        } else {
+            System.out.println("Введённое число нечётное");
+        }
+
+        // Управляющие конструкции. Задание 4
+        System.out.print("Введите значение температуры: ");
+        int temperature = sc.nextInt();
+        if (temperature > -5){
+            System.out.println("Тепло");
+        } else if(temperature > -20){
+            System.out.println("Нормально");
+        } else {
+            System.out.println("Холодно");
+        }
+
+        // Управляющие конструкции. Задание 5
+        System.out.print("Введите номер цвета радуги: ");
+        String color;
+        int colorNumber = sc.nextInt();
+        color = switch (colorNumber) {
+            case 1 -> "Красный";
+            case 2 -> "Оранжевый";
+            case 3 -> "Жёлтый";
+            case 4 -> "Зелёный";
+            case 5 -> "Голубой";
+            case 6 -> "Синий";
+            case 7 -> "Фиолетовый";
+            default -> "Введённое число не является номером цвета радуги";
+        };
+        System.out.println(color);
+
+        // Циклы. Задание 1
+        for(number = 1; number < 100; number++) {
+            if (number % 2 != 0) {
+                System.out.print(" " + number);
+            }
+        }
+
+        // Циклы. Задание 2
+        number = sc.nextInt();
+        int result = 0;
+        if (number >= 0) {
+            for (int sum = 2; sum < number; sum++) {
+                result += sum;
+            }
+            System.out.println(result);
+        }
+
+        // Циклы. Задание 3
+        int i = 0;
+        while(i<=100) {
+            i++;
+            if (i % 7 == 0) {
+                System.out.print(" " + i);
+            }
+        }
+
+        // Циклы. Задание 4
+        i = 10;
+        while(i <= 20) {
+            System.out.print(" " + i * i);
+            i++;
+        }
+
+        // Доплнительно. Задание 1
+        System.out.print("\nВведите желаемое количество выводимых членов последовательности Фибоначчи: ");
+        int n = sc.nextInt();
+        int number1 = 0;
+        int number2 = 1;
+        int number3;
+        System.out.print(number1 + " " + number2 + " ");
```

Comment:
What if n = 1?
Is this output will be correct?

Reply:
Corrected